### ISSUE TITLE
Is this the normal behavior of --resume get_last?

### ISSUE THREAD

**Issue by MarioDuran** [ultralytics/yolov5#744]
Hi! Thanks again for sharing such amazing work!

Since I´m working with Google Collab, I need to to be able to resume the training every day. 
in Yolov5 v1.0 this was the behavior of --resume get last. As can be visualized, it worked like a charm!

![YoloV5V1](https://user-images.githubusercontent.com/57765660/90298076-9ad54800-de45-11ea-8d2d-7cb17cde786d.png)

 but something changed in V2. and I got the following behavior. 

![YolovV5V2](https://user-images.githubusercontent.com/57765660/90298431-1f749600-de47-11ea-9d55-c5f9390c5b04.png)

It appears to be starting over, but now with I think slower learning rate (perhaps some variables are being resumed, but not all)

I used the next command for resuming both.

!python train.py --img 640 --batch 9 --epochs 500 --data ./data/coco_c.yaml --cfg ./models/yolov5xc.yaml --resume get_last --cache-images

Thank you in advance for any comment!

**Comment by glenn-jocher** [comment]
@MarioDuran in general, training only requires a single-source, either --cfg or --weights, but not both, unless you are trying to train a custom model.

I can't speak to resume, I would avoid it if you can as it's currently experimental only.

**Comment by glenn-jocher** [comment]
@MarioDuran I was forced into fixing the --resume functionality recently after a few runs shut down on our GCP VMs. I have a branch open that I'm running tests on, which should be merged soon at https://github.com/ultralytics/yolov5/tree/resume for PR https://github.com/ultralytics/yolov5/pull/756.

**Comment by glenn-jocher** [comment]
@MarioDuran #756 is merged. origin/master --resume functionality should now be operating correctly. Recommend you view usage examples on the PR.

**Comment by MarioDuran** [comment]
> @MarioDuran I was forced into fixing the --resume functionality recently after a few runs shut down on our GCP VMs. I have a branch open that I'm running tests on, which should be merged soon at https://github.com/ultralytics/yolov5/tree/resume for PR #756.

I´m sorry Glenn :( Hope it didn´t give you too much trouble. 

> @MarioDuran #756 is merged. origin/master --resume functionality should now be operating correctly. Recommend you view usage examples on the PR.

Wow! amazing Glenn! thank you so much, I´m going to test it right now! and tomorrow will let you know how it went :)

**Comment by MarioDuran** [comment]
![fixresume](https://user-images.githubusercontent.com/57765660/90690749-19a3f980-e227-11ea-97f4-608611b5547f.png)

Is working like a charm! :) thank you so much Glenn! hope it didn´t give you too much trouble!

**Comment by glenn-jocher** [comment]
@MarioDuran awesome, great!

One thing you could help with is to comment on PR https://github.com/ultralytics/yolov5/pull/765. The PR modifies --resume so that the same exact runs/exp folder is used, overwriting last.pt and best.pt, creating no new files/folders. 

It's implemented perfectly in the PR, but I'm unsure of whether to pull the trigger on it as I can't gauge what the majority expected behavior is. In your example above the most obvious difference would be that the resumed run would continue the orange line rather than making a new blue line, making for a more seamless training, but also not making it clear where the resumed run started, possibly making debugging resumed runs harder in the future. What do you think?
